internal/mailer/mailer.go

```diff
@@ -43,16 +41,7 @@ type EmailData struct {
 
 // NewMailer returns a new gotrue mailer
 func NewMailer(globalConfig *conf.GlobalConfiguration) Mailer {
-	mail := gomail.NewMessage()
-
-	mail.SetHeaders(map[string][]string{
```

Comment:
Do we not need the additional headers here? )e.g. the charset specification and the message-id